Applications/Preferences/Modules/Display/Display.m

```diff
@@ -42,7 +42,7 @@
 #import "AppController.h"
 #import "Display.h"
 
-NSString *NXTDesktopBackgroundColor = @"DesktopBackgroundColor";
+NSString *NXTDesktopBackgroundColor = @"NXDesktopBackgroundColor";
```

Comment:
Have you noticed error on saving background color? Changing name just creates new key in NXGlobalDomain and should not lead to any error. I'd like to hold new name (without NX prefix) and change default values (and key) in /etc/skel.

Reply:
I downloaded version 0.90 for Centos 7, to do the example. I see that the changes are not saved. No problem you can close this change, with another more satisfactory solution.

![example](https://user-images.githubusercontent.com/3104815/87979726-1f77b500-ca98-11ea-9d75-fc78197beda2.gif)

Reply:
@armm77 now I know where the problem resides (SystemKit uses hardcoded key). I'm going to merge this PR and move this definition into OSScreen class header file.

Reply:
@trunkmaster I made the rpms with the scripts, and the changes are not yet saved in plist.